Dockerfile

```diff
@@ -9,10 +9,12 @@ LABEL authors="Segelzwerg"
 FROM python:${PYTHON_VERSION}-slim
 WORKDIR /app
 COPY --from=poetry /app/requirements.txt .
+RUN apt-get update
+RUN apt-get install gettext -y
```

Comment:
do we need to delete the apt indices again to keep the image small?

Reply:
Yeah probably a good idea.